client/src/components/Review.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import StarReview from './StarReview.jsx';
+import { format } from "date-fns";
+
+const Review = ({review}) => {
+  return (
+    <React.Fragment>
```

Comment:
This React.Fragment looks redundant, since the div immediately below it is a single element.

Reply:
Yes, thank you. I was removing them from other similar areas and forgot this. I hadn't realized that it was for lists of elements. I'll remove in next commit.